notes_app/notes/management/commands/testdata.py

```diff
@@ -1,11 +1,35 @@
 from django.core.management.base import BaseCommand, CommandError
 from django.contrib.auth.models import User
+from django.contrib.auth.hashers import make_password
 from notes.models import Note
 import random
 import _randomdata as randomdata
 
 
+def gen_random_users(user_count):
+    """Puts a set of random users of given length into the db."""
+    username_tpl = "demo_user_{:0>2}"
+    first_name = "demo"
+    last_name = "demo"
+    email_tpl = "demo{}@example.com"
+    password = "demo"
+    for i in xrange(user_count):
+        try:
+            User.objects.get(username=username_tpl.format(i))
```

Comment:
to create sample data we have to specify the username explicitly and it happens to be a model key.

django documentation says this on get_or_create:
> The get_or_create() method has similar error behavior to create() when you’re using manually specified primary keys. If an object needs to be created and the key already exists in the database, an IntegrityError will be raised.